.github/workflows/ci-build-and-push.yml

```
@@ -0,0 +1,45 @@
+name: ci-build-and-push
+
+on:
+  push:
+    branches:
+      - main  
+  pull_request:
+    branches:
+      - main  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
```

run on kubearmor repo only
```suggestion
if: github.repository == 'kubearmor/kubearmor'
```